src/shared/lib/hooks/useFilledValue.ts

```diff
@@ -0,0 +1,20 @@
+import {UseFormSetValue} from "react-hook-form";
```

Comment:
This hook is created for the reset-password form. I don't think it can be reused anywhere, so you need to move it to features/authentication/reset-password/hooks

Reply:
I think that it can be reused later in filters to save them.